docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala

```diff
@@ -525,13 +524,17 @@ class HttpServerExampleSpec extends WordSpec with Matchers
 
     object WebServer {
 
-      case class Bid(userId: String, bid: Int)
+      case class Bid(userId: String, offer: Int)
       case object GetBids
       case class Bids(bids: List[Bid])
 
       class Auction extends Actor {
+        var bids = List.empty[Bid]
         def receive = {
-          case Bid(userId, bid) => println(s"Bid complete: $userId, $bid")
+          case bid @ Bid(userId, offer) =>
+            bids = bids :+ bid
+            println(s"Bid complete: $userId, $offer")
+          case GetBids => sender() ! Bids(bids)
           case _ => println("Invalid message")
```

Comment:
I know it was already a print, but could you use the Actor System's log instead?